langchain4j-agentic/src/main/java/dev/langchain4j/agentic/planner/Planner.java

```diff
@@ -32,4 +34,9 @@ default Action done(Object result) {
         return new Action.DoneWithResultAction(result);
     }
 
+    default void checkSubAgents(List<AgentInstance> subAgents, AgentInstance plannerAgent) {
```

Comment:
As I wrote, I would prefer to avoid having this method on the `Planner` interface, please see https://github.com/langchain4j/langchain4j/pull/3878/files#r2526711521